src/Slider.tsx

```diff
@@ -16,6 +16,9 @@ interface Props {
   className?: string;
   label: string;
   value: number;
+  /**
+   * Event handler called when the value changes during an interaction.
+   */
   onValueChange: (value: number) => void;
```

Comment:
Should this be made optional to match the `?.` below?

Reply:
I think that's actually a mistake. We need to always handle change as otherwise the value of the slider doesn't update on input and then you can't actually move it. So change is always required but commit is optional.

I've updated the usage of `?.`